src/progressive_automations_python/prefect_flows.py

```diff
@@ -0,0 +1,235 @@
+"""
+Simplified Prefect flows for automated desk control.
+
+Provides scheduled automation and workflow orchestration using Prefect.
+Directly decorates core functions from desk_controller for task execution.
+"""
+
+import time
+from prefect import flow, task
+from prefect.logging import get_run_logger
+
+from progressive_automations_python.desk_controller import (
+    move_to_height, 
+    check_duty_cycle_status_before_execution
+)
+from progressive_automations_python.testing import (
+    test_sequence,
+    execute_custom_movements
+)
+from progressive_automations_python.config import LOWEST_HEIGHT
+from progressive_automations_python.duty_cycle import get_duty_cycle_status, load_state
+
+# Decorate core functions as tasks
+move_to_height_task = task(move_to_height)
+test_sequence_task = task(test_sequence)
+execute_custom_movements_task = task(execute_custom_movements)
+check_duty_cycle_status_task = task(check_duty_cycle_status_before_execution)
+
+
+# =============================================================================
+# FLOWS
+# =============================================================================
+
+@flow
+def simple_movement_flow(target_height: float):
+    """Simple Prefect flow for moving to a specific height with duty cycle checking"""
+    logger = get_run_logger()
+    logger.info(f"=== SIMPLE MOVEMENT FLOW ===")
+    logger.info(f"Target: {target_height}\"")
+    
+    # Check duty cycle status
+    initial_status = check_duty_cycle_status_task()
+    
+    # Abort if insufficient capacity
+    if initial_status["remaining_capacity"] < 1.0:
+        logger.error("❌ MOVEMENT ABORTED: Insufficient duty cycle capacity")
```

Comment:
Shouldn't this be based on what's requested? Not just an arbitrary 1.0? Also, if that's the case, that just means you need to wait until it's ready, and I'm pretty sure you can calculate that info of the wait time required before it can move (i.e., the estimated time to completion of the move)